frontend/src/pages/Settings.tsx

```diff
@@ -0,0 +1,49 @@
+import Header from "../components/Header";
+import { ToggleButton } from "../components/UtilityComponents";
+
+function Settings() {
+  return (
+    <>
+      <Header>Settings</Header>
```

Comment:
Is it possible to create this `Header` component in one place like the `NavBar`?